themes/bootstrap3/templates/record/checkbox.phtml

```diff
@@ -1,5 +1,6 @@
 <label class="record-checkbox hidden-print">
   <input class="checkbox-select-item" type="checkbox" name="ids[]" value="<?=$this->escapeHtmlAttr($this->id) ?>"<? if(isset($this->formAttr)): ?> form="<?=$this->formAttr ?>"<? endif; ?>/>
   <span class="checkbox-icon"></span>
+  <?=$this->count ?>
```

Comment:
If $number defaults to null in the getCheckbox function, and we're displaying the count value to avoid empty labels, it seems that we haven't fully solved the problem here. Would it be better to display an sr-only text in here, like "checkbox for record x"? I realize that might require a new translation, in which case it might be better to target this PR against master rather than release-4.1.

Reply:
How about `aria-label="Select record x"`?

Reply:
I didn't see any translations that were a good fit and I didn't want to add one. `aria-label` does seem like a good solution though.

Reply:
If we're going to go for a new translation, maybe it should be `Select record <title>` since tying a control to a title is probably better than to a result number -- though the better approach may depend on how this plays out in an actual screen reader. If we cause the same title to get read over and over again, we could be really slowing things down and causing annoyance. Is there a precedent in the wild we can look at for inspiration?

Reply:
I don't know any precedent. I think maybe `Check result number <#>` might be a more compact and clear identifier. I agree that the numbers aren't visually necessary.

Reply:
I'd use "select" instead of "check", but then I'm not native English speaker...

Reply:
I agree on "Select" -- "check" could be more easily misunderstood to mean "confirm/investigate" rather than "select."